emperor/support_files/js/ui-state.js

```diff
@@ -1,4 +1,4 @@
-define(['three', 'underscore'], function(THREE, _) {
+define(['three'], function(THREE) {
```

Comment:
Just to confirm, underscore is not used anymore, right?

Reply:
You are correct, I just checked again and there's no matches to "_".